src/Asset/AssetDefinition.php

```diff
@@ -141,12 +141,31 @@ public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $
     public function showForm($ID, array $options = [])
     {
         $this->initForm($ID, $options);
+        $options['candel'] = false;
+
+        if (!self::isNewID($ID)) {
+            // Add custom delete button that will show the number of assets using this definition and a link to their search page
+            $asset_count = countElementsInTable('glpi_assets_assets', ['assets_assetdefinitions_id' => $ID]);
```

Comment:
We should probably not count templates. Also, maybe we should not count items from trash bin, but I am not sure of that.